packages/wonder-blocks-tooltip/components/tooltip.integration.test.js

```diff
@@ -42,14 +45,50 @@ describe("tooltip integration tests", () => {
         const anchor = anchorWrapper.getDOMNode();
         // start hovering
         anchor && anchor.dispatchEvent(new FocusEvent("mouseenter"));
+        anchor && anchor.dispatchEvent(new FocusEvent("mouseleave"));
         expect(wrapper.state("timeoutID")).toEqual(expect.any(Number));
 
+        const bubbleTimeoutCheck = (callback) =>
+            setTimeout(() => {
+                expect(wrapper.state("timeoutID")).toEqual(expect.any(Number));
```

Comment:
Instead of doing the `expect` inside of the timeout, a more reliable approach would be to add a local variable to the test and set the value inside the timeout and move the `expect` to the bottom of the the test.

Reply:
```
expect(wrapper.state("timeoutID")).toEqual(expect.any(Number));
```
was asserted on line 49, shouldn't that be sufficient?